packages/wme-ui/src/components/setup-card-accordion/setup-card-accordion.tsx

```diff
@@ -52,13 +53,15 @@ export default function SetupCardAccordion(props: SetupCardAccordionProps) {
     chipBackground,
     chipText,
     isComplete,
+    toggleOpen,
     ...rest
   } = props;
 
   return (
     <SetupCard>
       <StyledSetupCardAccordion className="WmeSetupCardAccordion-root" {...rest}>
         <AccordionSummary
+          {... toggleOpen ? { onClick: () => toggleOpen(id) } : {}}
```

Comment:
Interesting convention, here. I don't think I've seen a spread operator used in this context before.

Reply:
It works really well when the prop changes the behavior of the element. In this case, if onClick is declared, the accordion losses the built-in open toggling. So I had to avoid adding it to the element when the parent wasn't controlling the toggling.